v2/samples/keyvault/v1beta/v1beta20210401preview_vault.yaml

```diff
@@ -31,15 +33,3 @@ spec:
             - "get"
           storage:
             - "get"
-      - applicationId: "1C793267-c310-d4ae-7BD5-5Af5BEF875D3"
```

Comment:
Is this not required for the test to record successfully? For some reason I thought it was

Reply:
I thought it was just a pair of access policies, one from config, the other hard coded. The recording did work.